charts/localstack/templates/deployment.yaml

```diff
@@ -119,8 +123,14 @@ spec:
               value: {{ .Values.lambda.environment_timeout | quote }}
             {{- end }}
             {{- if .Values.persistence.enabled }}
+            - name: DATA_DIR
```

Comment:
This line is not within the conditional (it's duplicated and should be removed).